### PR TITLE
Native OpenTok SDK 2.28.0 features

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java [hunk 1]
```diff
@@ -421,6 +424,16 @@ public void setVideoTransformers(String publisherId, ReadableArray videoTransfor
         }
     }
 
+    @ReactMethod
+    public void setAudioTransformers(String publisherId, ReadableArray audioTransformers) {
+        ConcurrentHashMap<String, Publisher> mPublishers = sharedState.getPublishers();
+        Publisher mPublisher = mPublishers.get(publisherId);
+        if (mPublisher != null) {
+          ArrayList<AudioTransformer> nativeAudioTransformers = Utils.sanitizeAudioTransformerList(mPublisher, audioTransformers);
```
Comment (marinaserranomontes) [inline android/src/main/java/com/opentokreactnative/OTSessionManager.java:432]: @jeffswartz should we run an exception if list is empty?

File: @types/index.d.ts [hunk 2]
```diff
@@ -224,6 +224,11 @@ declare module "opentok-react-native" {
         credential?: string;
       }[];
     };
+
+    /**
+     * Enable single peer connection for the client.
+     */
+    enableSinglePeerConnection?: boolean;
```
Comment (marinaserranomontes) [inline @types/index.d.ts:231]: is it false by default?

File: android/src/main/java/com/opentokreactnative/OTSessionManager.java (hunk 1 above)
Comment (jeffswartz) [reply android/src/main/java/com/opentokreactnative/OTSessionManager.java:432]: @marinaserranomontes No. Passinig in an empty array is the way to clear audio transformers (per the docs).

**Review by marinaserranomontes — APPROVED** [review]
LGTM

**Comment by marinaserranomontes** [comment]
Should we bump version to v2.28.1?

**Pull request merged**